apps/ui/src/components/ProposalExecutionActions.vue

```diff
@@ -7,70 +7,20 @@ import { Proposal as ProposalType } from '@/types';
 const props = defineProps<{ proposal: ProposalType }>();
 
 const { web3 } = useWeb3();
-const { finalizeProposal, executeTransactions, executeQueuedProposal, vetoProposal } = useActions();
-
-const finalizeProposalSending = ref(false);
-const executeTransactionsSending = ref(false);
-const executeQueuedProposalSending = ref(false);
-const vetoProposalSending = ref(false);
-const currentTimestamp = ref(Date.now());
+const {
+  hasFinalize,
+  finalizeProposalSending,
+  executeProposalSending,
+  executeQueuedProposalSending,
+  vetoProposalSending,
+  executionCountdown,
+  finalizeProposal,
+  executeProposal,
+  executeQueuedProposal,
+  vetoProposal
+} = useExecutionActions(props.proposal);
 
 const network = computed(() => getNetwork(props.proposal.network));
-const baseNetwork = computed(() =>
```

Comment:
Not using `baseNetwork` anymore ?

Reply:
No, we don't have use-case for it anymore as all current contracts have execution on same chain as proposal itself.
EthRelayer will add cross-chain execution, but in that case `execution_tx` is on the same chain anyway. L1 execution will be fetched differently, so it will need to be set displayed differently as well.